etc/listproviders/event.status.properties

```diff
@@ -0,0 +1,9 @@
+list.name=eventStatusFilter
```

Comment:
Thanks for factoring out the hard-coded lists, but as for this list, I don't think that it is of any use. Please remove the list "eventStatusFilter" or provide a reason why you think it is needed.